main.js

```diff
@@ -47,15 +44,20 @@ function validateUserInput() {
   }
 }
 
-
 function displayCards() {
   ideaCardGrid.innerHTML = "";
   for (var i = 0; i < ideas.length; i++) {
+    var starSource;
+    if (ideas[i].star) {
+      starSource = "./assets/star-active.svg";
+    } else {
+      starSource = "./assets/star.svg";
+    }
     ideaCardGrid.innerHTML +=
     `<div class="box-container" id="${ideas[i].id}">
       <div class="box-header-container">
-      <input class="star-btn" type="image" name="star button" src="./assets/star.svg" alt="picture-of-a-star">
-      <input class="delete-btn" type="image" name="delete button" src="./assets/delete.svg" alt="picture-of-an-x">
+        <input class="star-btn" type="image" name="star button" src="${starSource}" alt="picture-of-a-star">
+        <input class="delete-btn" type="image" name="delete button" src="./assets/delete.svg" alt="picture-of-an-x">
```

Comment:
I created a starSource variable which assigns the appropriate image source depending on the star property of the idea.